.github/workflows/e2e.yml

```diff
@@ -33,11 +33,13 @@ jobs:
         path: |
           target/
         key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+    - uses: actions/setup-node@v4
+      with:
+        node-version: '20.x'
```

Comment:
[same suggestion](https://github.com/stellar/js-stellar-sdk/pull/1151/files#r2010625822) for consideration of using an env variable for version